winlogbeat/docs/fields.asciidoc

```diff
@@ -74,11 +75,11 @@ required: True
 The name of the computer that generated the record. When using Windows event forwarding, this name can differ from the `beat.hostname`.
 
 
-==== eventCategory
+==== category
 
 type: string
 
-required: True
+required: False
```

Comment:
I will add a changelog entry. I did make a change here. Previously the Beat used "Unknown" as the value when the data could not be obtained, now it just omits the field from the event.